scene/gui/rich_text_label.cpp

```diff
@@ -4079,6 +4079,70 @@ void RichTextLabel::parse_bbcode(const String &p_bbcode) {
 	append_text(p_bbcode);
 }
 
+int RichTextLabel::_find_unquoted(const String &p_src, char32_t p_chr, int p_from) {
+	if (p_from < 0) {
+		return -1;
+	}
+
+	const int len = p_src.length();
+	if (len == 0) {
+		return -1;
+	}
+
+	const char32_t *src = p_src.get_data();
+	bool in_squote = false;
+	bool in_dquote = false;
+	for (int i = p_from; i < len; i++) {
+		if (in_dquote) {
+			if (src[i] == 0x0022) {
```

Comment:
You could use single-quoted char instead of code, no?
Or at least put a comment what the code is for.